packages/framework-types/src/concepts/authorizers.ts

```diff
@@ -1,8 +1,10 @@
-import { EventSearchRequest, ReadModelRequestEnvelope, UserEnvelope, CommandEnvelope } from '../envelope'
+import { EventSearchRequest, ReadModelRequestEnvelope, UserEnvelope, CommandEnvelope, QueryEnvelope } from "../envelope";
```

Comment:
remove `;` and replace double quotes by single quotes.

Reply:
It looks like my code is the one without the `;` and with single quotes. Isn't it?

Reply:
oh sorry! let me fix it